functions.py

```diff
@@ -1,84 +1,133 @@
+from command_factory import *
+from ICommand import ICommand
 import json
 import os
+import keys
 
 
-def search(path, primary_key):
-    y = False
-    for r, d, f in os.walk(path):
-        if primary_key in f:
-            y = True
-            break
-    return y
-
-
-def creates_dir(schema):
-    x = json.load(open(schema, 'r'))
-    p = os.path.join(os.getcwd(), x['database_name'])
-    os.mkdir(p)
-    for i in x['Tables']:
-        d = os.path.join(p, i['name'])
-        os.mkdir(d)
-
-
-def creates(schema, table, primary_key):
-    x = json.load(open(schema, 'r'))
-    path = os.getcwd() + '\\' + x['database_name'] + '\\' + table + '\\' + primary_key
-    f = open(path, 'w')
-    f.write("{\n")
-    i = 0
-    for k in x['Tables']:
-        if k['name'] != table:
-            i = i + 1
+
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):

```

Comment:
is this schema or schema_url ?

Reply:
schema file

Reply:
The name is very confusing. 
schema --> means it is an object or map 
schema_file --> means it is a File object 
schema_path --> means it is a url. 

When someone read the code, he will get confused.

Naming variables are very very important